hols/todo/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
We should simplify the commands and eventually inline them in a README.

Reply:
done